src/pkg/cluster/zarf.go

```diff
@@ -52,7 +52,11 @@ func (c *Cluster) GetDeployedZarfPackages(ctx context.Context) ([]types.Deployed
 		deployedPackages = append(deployedPackages, deployedPackage)
 	}
 
-	return deployedPackages, errors.Join(errs...)
+	// REVIEW(mkcp): This seems like it could be a breaking behavior change. Can I get a second set of eyes here?
```

Comment:
Review request

Reply:
This is not breaking as far as I can see. When calling `errors.Join` it will return nil if the slice given is empty. Here is a suggestion for how I would implement this.

```golang
err = errors.Join(errs)
if err != nil {
  return nil, err
}
return deployedPackages, nil
```

Reply:
huh, TIL! resolved in recent commit